client/src/app/submission/submission.component.html

```diff
@@ -28,6 +28,39 @@ <h2 class="vf-text vf-text-heading--2">
   <div>
     <div class="vf-text-body vf-text-body--2">{{projectTitle}}</div>
     <div class="vf-text-body vf-text-body--3">{{getContributors(project)}}</div>
+    <div *ngIf="validationSummary">
+      <p>Your metadata validation returned <span class="error-text">{{validationSummary.totalInvalid}} error(s)</span>. Review and fix them below.</p>
```

Comment:
remove 'metadata', as totalInvalid may include both data and metadata validation.